### PR TITLE
Fix missing VERSION file in buildbot_worker wheel package.

### PR DESCRIPTION
**Pull request by pmisik**
Bug #5948, #4464

## Contributor Checklist:

* [ ] I have updated the unit tests
* [x] I have created a file in the `master/buildbot/newsfragments` directory (and read the `README.txt` in that directory)
* [ ] I have updated the appropriate documentation

### CODE REVIEW

**Comment by tardyp** [comment]
I am not sure why this worked, and do not work anymore.

Should we include the VERSION file in the MANIFEST.in?

**Comment by pmisik** [comment]
> 
> 
> I am not sure why this worked, and do not work anymore.
> 
> Should we include the VERSION file in the MANIFEST.in?

I think problem is that VERSION file is generated during packing.
`our_sdist` is called while sdist is packed but not while `bdist_wheel` command is running.

It seems revert of b339c0ede7876d301415b7b875e33cfc496448c1 solves the problem.

**Comment by tardyp** [comment]
what worries me is that this solution is different from what is in the other packages.

We should have similar logic everywhere.

I think that revert  b339c0e leads to that actually as I see that master package has same package tweak that wasn't removed
install_data method, is probably the method that works the best. I was wrong in trying to remove it.

@p12tic  any opinion?

File: worker/setup.py [hunk 1]
```diff
@@ -33,12 +33,30 @@
...
     from setuptools.command.sdist import sdist
+    from distutils.command.install_data import install_data
 except ImportError:
     setuptools = None
     from distutils.command.sdist import sdist
     from distutils.core import setup
 
 
+class our_install_data(install_data):
+
```
Comment (tardyp) [inline worker/setup.py:44]: lgtm

**Review by tardyp — APPROVED** [review]

**Comment by p12tic** [comment]
I think that it makes sense to keep what master is doing. So this PR is in good direction.

**Review by p12tic — APPROVED** [review]

**Pull request merged**